modules/installation-approve-csrs.adoc

```diff
@@ -63,9 +63,9 @@ Because the CSRs rotate automatically, approve your CSRs within an hour
 of adding the machines to the cluster. If you do not approve them within an
 hour, the certificates will rotate, and more than two certificates will be
 present for each node. You must approve all of these certificates. After you
-approve the initial CSRs, the subsequent CSRs are automatically approved by the
-cluster `kube-controller-manager`. You must implement a method of automatically
-approving the kubelet serving certificate requests.
+approve the initial CSRs, the subsequent node client CSRs are automatically
```

Comment:
@russellb, what happens to the server request CSRs?

Reply:
they must be manually approved, or some automation created to approve them.  That's what the following sentence refers to.  I was just trying to make it clear that the client certs will be auto approved, but not the server certs.  does that help?

Reply:
That does. Thanks!